_posts/0000-01-03-Manuel.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: slide
+title: "Manuel"
+---
+
+I'm torn between JavaScript and C#
+I love the flexibility of JavaScript but at the same time I don't like having to deal with browser cross-compatibility and polyfills.
```

Comment:
For me `JavaScript` is great because we can build a whole application, like backend and frontend.